### DISCUSSION TITLE
Does repack works fine on Windows?

### DISCUSSION THREAD

**Discussion by ViniciusUrias** [callstack/repack#339] [Q&A]
I was using windows 10, but all repack/module-federation examples were crashing.
Errors like:
'Can't find __webpack_require__';
'[ERR_UNSUPPORTED_ESM_URL_SCHEME]';
'Unhandled error. ([ScriptManager] Failed to load script:)';

Then I upgraded to windows 11 and the same errors still ocurring.
I think the last solution is change to some linux distro.
Obs: I tested it like 100 times before coming here, used 3 different versions of node, react and react-native, tested with .mjs, .cjs and .js, created a brand new app to configure, cloned the existing examples and I simply can't find a solution

**Comment by alekgomes** [comment]
Estou com erros de `ERR_UNSUPPORTED_ESM_URL_SCHEME]` por aqui também, usando w11

**Comment by ElielC** [comment]
Re.pack versions 5.1.x should work fine
